.gitignore

```diff
@@ -5,6 +5,10 @@
 ## User settings
 xcuserdata/
 
+## Files
+keys.plist
+*.DS_Store
```

Comment:
Please delete all DS_Store files in the repository as it currently stands (https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository)